### PR TITLE
add description prop to relation model

### PR DESCRIPTION
**Pull request by itamar-smirra-port**
# Description

What - Adding description to relations

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

**Review by erikzaadi — APPROVED** [review]

File: port/blueprint/model.go [hunk 1]
```diff
@@ -102,10 +102,11 @@ type PropertiesModel struct {
...
 type RelationModel struct {
-	Target   types.String `tfsdk:"target"`
-	Title    types.String `tfsdk:"title"`
-	Required types.Bool   `tfsdk:"required"`
-	Many     types.Bool   `tfsdk:"many"`
+	Target   	types.String `tfsdk:"target"`
+	Title    	types.String `tfsdk:"title"`
+	Description types.String `tfsdk:"description"`
+	Required 	types.Bool   `tfsdk:"required"`
+	Many     	types.Bool   `tfsdk:"many"`
```
Comment (Tankilevitch) [inline port/blueprint/model.go:109]: seems like something with you linting is not configured correctly

File: internal/cli/models.go [hunk 2]
```diff
@@ -318,11 +318,12 @@ type (
...
 
 	Relation struct {
-		Identifier *string `json:"identifier,omitempty"`
-		Title      *string `json:"title,omitempty"`
-		Target     *string `json:"target,omitempty"`
-		Required   *bool   `json:"required,omitempty"`
-		Many       *bool   `json:"many,omitempty"`
+		Identifier 		*string `json:"identifier,omitempty"`
+		Title      		*string `json:"title,omitempty"`
+		Description     *string `json:"description,omitempty"`
```
Comment (Tankilevitch) [inline internal/cli/models.go:323]: same here

File: port/blueprint/updatePropertiesToState.go [hunk 3]
```diff
@@ -177,10 +177,11 @@ func addRelationsToState(b *cli.Blueprint, bm *BlueprintModel) {
 		}
 
 		relationModel := &RelationModel{
-			Target:   types.StringValue(*v.Target),
-			Title:    flex.GoStringToFramework(v.Title),
-			Many:     flex.GoBoolToFramework(v.Many),
-			Required: flex.GoBoolToFramework(v.Required),
+			Target:   		types.StringValue(*v.Target),
+			Title:    		flex.GoStringToFramework(v.Title),
+			Description:    flex.GoStringToFramework(v.Description),
```
Comment (Tankilevitch) [inline port/blueprint/updatePropertiesToState.go:182]: same here 🤔  I wonder why linting doesn't fail for that @erikzaadi ?

**Pull request merged**